src/account-info/AccountDetailsScreen.js

```diff
@@ -16,17 +16,21 @@ class AccountDetailsScreen extends Component {
     avatarUrl: string,
   }
 
+  componentWillMount() {
+    this.email = this.props.email;
```

Comment:
Cool can we add that as a comment here? Would be useful to someone reading the code in the future

I don't really like this pattern -- we should revisit later and see if there's a way to avoid having to store state in this component